packages/split-view/README.md

```diff
@@ -197,4 +249,4 @@ import { SplitView } from '@spectrum-web-components/split-view';
 
 ## Accessibility
 
```

Comment:
A few other accessibility features you could mention here:

The splitter:

- has `role="separator"` set on it
- sets `aria-controls` when resizable, pointing to the first pane's ID
- sets `aria-orientation` to horizontal or vertical
- uses `aria-valuenow` (honestly had to look that one up)

Also the keyboard navigation is kinda cool here! I believe you can use arrow keys or page up/down to move the resizable splitter, but page up/down increment it at a greater value than the arrow keys (10px vs. 50px), and home/end will send the splitter to the min/max position.